wagtail/admin/templates/wagtailadmin/workflows/workflow_status.html

```diff
@@ -1,67 +1,88 @@
 {% load wagtailadmin_tags i18n %}
 
-{% include "wagtailadmin/shared/header.html" with title=workflow_state.workflow.name icon="tasks" %}
+<div>
+    {# Timeline #}
+    <ul class="workflow-timeline">
 
-<div class="nice-padding">
-    {% if page.get_latest_revision %}
-        <p>
-            {% blocktrans trimmed with workflow_name=workflow_state.workflow.name started_at=workflow_state.created_at %}
-                Submitted to <em>{{ workflow_name }}</em> at {{ started_at }}
-            {% endblocktrans %}
-            {% if workflow_state.requested_by %}
-                {% blocktrans trimmed with modified_by=workflow_state.requested_by|user_display_name %}by {{ modified_by }}{% endblocktrans %}
-                <span class="avatar small"><img src="{% avatar_url page.get_latest_revision.user size=25 %}" alt="" /></span>
-            {% endif %}
-        </p>
-    {% endif %}
+        {# Submitted status #}
+        <li class="workflow-timeline__item !w-text-positive-100">
+            <div class="workflow-timeline__line"></div>
+            {% icon name="circle-check" class_name="workflow-timeline__icon" %}
+            {% trans "Submitted" %}
+        </li>
 
-    <ul class="workflow-timeline">
-        <li class="approved">{% icon name="success" class_name="default" %}{% trans "Submitted" %}</li>
+        {# Workflow tasks #}
         {% trans "Not started" as not_started_text %}
-        {% for task in workflow_tasks %}
-            <li data-wagtail-tooltip="{{ task.name }}: {{ task.task_state.get_status_display|default:not_started_text }}" {% if task.task_state and task.task_state.status %}class="{{ task.task_state.status }}"{% endif %}>
-                {% if task.task_state.status == "rejected" %}
-                    {% icon name="warning" class_name="default" %}<strong>{{ task.name }}</strong>
+        {% if workflow_state %}
+            {% for task in workflow_state.all_tasks_with_state %}
+                {% with status=task.task_state.status %}
+                    <li aria-label="{{ task.name }}: {{ task.task_state.get_status_display|default:not_started_text }}"
```

Comment:
`aria-label` isn’t widely supported on non-interactive elements like `li`, so we’ll need to move this to the `icon`’s title.